### PR TITLE
OHRM5X-624: Initiate leave entitlement usage report APIs

### PR DESCRIPTION
**Pull request by RajithaKumara**

### CODE REVIEW

**Comment by RajithaKumara** [comment]
@samanthajayasinghe can you review this
EmployeeLeaveEntitlementUsageReport
Header
LeaveReportAPI
LeaveReportDataAPI

File: symfony/plugins/orangehrmLeavePlugin/Report/EmployeeLeaveEntitlementUsageReport.php [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+                (new Column(self::PARAMETER_ENTITLEMENT_DAYS))->setName('Leave Entitlements (Days)'),
+                (new Column(self::PARAMETER_PENDING_APPROVAL_DAYS))->setName('Leave Pending Approval (Days)'),
+                (new Column(self::PARAMETER_SCHEDULED_DAYS))->setName('Leave Scheduled (Days)'),
+                (new Column(self::PARAMETER_TAKEN_DAYS))->setName('Leave Taken (Days)'),
+                (new Column(self::PARAMETER_BALANCE_DAYS))->setName('Leave Balance (Days)'),
+            ]
+        );
+    }
+
+    public function getFilterDefinition()
```
Comment (samanthajayasinghe) [inline symfony/plugins/orangehrmLeavePlugin/Report/EmployeeLeaveEntitlementUsageReport.php:64]: Is it still needed?

**Comment by samanthajayasinghe** [comment]
@RajithaKumara what is the difference between `Column` and `StackColumn` ?

**Comment by RajithaKumara** [comment]
> @RajithaKumara what is the difference between `Column` and `StackColumn` ?

Main usage for group the column,
https://revolist.github.io/revogrid/guide/column.grouping.html

File: symfony/plugins/orangehrmLeavePlugin/Report/EmployeeLeaveEntitlementUsageReport.php (hunk 1 above)
Comment (RajithaKumara) [reply symfony/plugins/orangehrmLeavePlugin/Report/EmployeeLeaveEntitlementUsageReport.php:64]: We can have it to define filter component name, or we can remove

**Pull request merged**